build/docker-images/HealthChecks.UI.Image/Dockerfile

```diff
@@ -37,3 +37,4 @@ FROM base AS final
 WORKDIR /app
 COPY --from=publish /app .
 ENTRYPOINT ["dotnet", "HealthChecks.UI.Image.dll"]
+CMD []
```

Comment:
Adding `CMD []` in the Dockerfile will allow arguments to be used when running the Docker image. This will allow, for example, enabling Kubernetes Service Discovery by adding the argument `--HealthChecksUI:KubernetesDiscoveryService:Enabled=true` to the Pod spec arguments or `docker run` command. Without this configuration must be overridden through environment variables, ex:
``` yaml
- name: HealthChecksUI__KubernetesDiscoveryService__Enabled
  value: "true"
```